crates/primitives/src/block.rs

```diff
@@ -17,7 +17,6 @@ pub use reth_rpc_types::{
 #[derive(
     Debug, Clone, PartialEq, Eq, Default, Serialize, Deserialize, RlpEncodable, RlpDecodable,
 )]
-#[derive_arbitrary(rlp, 25)]
```

Comment:
when removing an arbitrary derive, we should make sure to re-add `add_arbitrary_tests(rlp)`